### ISSUE TITLE
left side off centered

### ISSUE THREAD

**Issue by DonPingus** [surfbryce/beautiful-lyrics#327]
The whole left side with the album and player is off centered and overlaps.
![beautiful lyrics](https://github.com/surfbryce/beautiful-lyrics/assets/74151712/56818845-89b1-406d-be77-6596ea5bc7e2)

**Comment by surfbryce** [comment]
@DonPingus Are you using any themes or is this just stock Spotify with the extension? Do you have anything else installed?

**Comment by DonPingus** [comment]
> @DonPingus Are you using any themes or is this just stock Spotify with the extension? Do you have anything else installed?

No themes just stock spotify with the extension and i have a snippet installed
![snippet](https://github.com/surfbryce/beautiful-lyrics/assets/74151712/f81b5faf-642f-403a-9fca-802bbff603e7)

**Comment by surfbryce** [comment]
@DonPingus does this still happen if you restart spotify?

**Comment by DonPingus** [comment]
> @DonPingus does this still happen if you restart spotify?

Yeah I have also tried redownloading spotify and made sure spicetify is up to date

**Comment by surfbryce** [comment]
@DonPingus Can you show me what Spotify looks like normally? As in, just a screenshot of Spotify when you're on the home screen, thanks!

**Comment by DonPingus** [comment]
> @DonPingus Can you show me what Spotify looks like normally? As in, just a screenshot of Spotify when you're on the home screen, thanks!

![spotifyhome](https://github.com/surfbryce/beautiful-lyrics/assets/74151712/32f921f3-c053-4bec-9d09-9cebf4ec6b28)

**Comment by surfbryce** [comment]
@DonPingus Ah seems like you have the new spotify UI they're testing out, I'll see if I can enable it. I think that is what is causing the glitchy look!

**Comment by DonPingus** [comment]
> @DonPingus Ah seems like you have the new spotify UI they're testing out, I'll see if I can enable it. I think that is what is causing the glitchy look!

Thank you very much I appreciate it :)

**Comment by 209726742** [comment]
Hi, I was wondering how I could use the UI interface shown above? Mine looks like the one below.
<img width="1106" alt="1561cec739a0e9be616c837c409485c" src="https://github.com/surfbryce/beautiful-lyrics/assets/54384262/2d4d8a4d-4cc4-4b06-b7dd-0523a97049df">
